src/lib/components/paginationWithLimit.svelte

```
@@ -8,24 +8,37 @@
+        pageParam = 'page',
+        removeOnFirstPage = false,
+        ...restProps
     }: {
         limit: number;
         offset: number;
         total: number;
         name: string;
         useCreateLink?: boolean;
+        pageParam?: string;
+        removeOnFirstPage?: boolean;
+        [key: string]: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)